cluster-autoscaler/cluster_autoscaler.go

```diff
@@ -16,5 +16,18 @@ limitations under the License.
 
 package main
 
+import (
+	"flag"
+	"fmt"
+	"k8s.io/contrib/cluster-autoscaler/config"
+)
+
+var (
+	migConfig config.MigConfigFlag
+)
+
 func main() {
+	flag.Var(&migConfig, "nodes", "sets min,max size and url of a MIG to be controlled by Cluster Autoscaler")
```

Comment:
Please explain that it's possible to use this flag multiple times to pass control more MIGs.